src/NServiceBus.AcceptanceTests/ApiExtension/When_extending_sendoptions.cs

```diff
@@ -5,7 +5,7 @@
     using AcceptanceTesting;
     using EndpointTemplates;
     using Extensibility;
-    using Pipeline;
+    using NServiceBus.Pipeline;
```

Comment:
Correct, since we put usings inside the NS we don't qualify the "sub NS"
usings.

On Sat, Mar 12, 2016 at 12:40 AM, Brandon Ording <notifications@github.com>
wrote:

> In
> src/NServiceBus.AcceptanceTests/ApiExtension/When_extending_sendoptions.cs
> <https://github.com/Particular/NServiceBus/pull/3579#discussion_r55905526>
> :
>
> > @@ -5,7 +5,7 @@
> >      using AcceptanceTesting;
> >      using EndpointTemplates;
> >      using Extensibility;
> > -    using Pipeline;
> > +    using NServiceBus.Pipeline;
>
> What is the desired format for using statements? I prefer to be explicit
> like this, but I think a new setting was introduced remove this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3579/files#r55905526>.
>